samples/client/others/rust/reqwest-oneof-discriminator-test/tests/enums.rs

```diff
@@ -0,0 +1,18 @@
+use reqwest_oneof_discriminator::models::FruitOneOfEnumMappingDisc;
+
+#[test]
+fn test_oneof_discriminator_deserialize() {
```

Comment:
I think it would be useful to start adding tests like this to the Rust generators, since just generating a sample project doesn't prevent future regressions.